src/AdminConsole/Components/Pages/App/Settings/SettingsComponents/ApiKeysSection.razor

```diff
@@ -11,84 +11,89 @@
 {
     <Panel Header="Api Keys">
         <p>To learn more about API key management, read more <a class="link-blue underline" href="https://docs.passwordless.dev/guide/admin-console/applications.html#api-key-management" target="_blank">here</a>. To delete an API key, first lock the API key.</p>
-    
+
         <EditForm class="flex" Model="CreateForm" FormName="@(CreateApiKeyFormName)" OnSubmit="OnCreateFormSubmitted">
             <button id="create-public-key-btn" class="btn-primary" name="CreateForm.Type" type="submit" value="public">Create Public Key</button>
             <button id="create-secret-key-btn" class="btn-primary ml-4" name="CreateForm.Type" type="submit" value="secret">Create Secret Key</button>
         </EditForm>
-    
-        <Table ColumnHeaders="@(new []{ "Api Key", "Created At", "Type", "Locked", "Scopes", string.Empty })" EmptyMessage="This application has no API Keys or API Secrets.">
-            @if (ApiKeys != null)
-            {
-                @foreach (var apiKey in ApiKeys)
+
+        <ConfirmEditForm Model="ConfirmedSelectedForm" FormName="@(ConfirmedSelectedApiKeyFormName)" name="@ConfirmedSelectedApiKeyFormName" OnSubmit="OnSelectedFormConfirmed">
```

Comment:
Blazor doesn't like multiple events with the same form name. So we have to wrap the entire table in the form similar to how we manage admins.